public/app/features/alerting/unified/rule-list/RuleList.v2.tsx

```diff
@@ -4,28 +4,29 @@ import { useLocation } from 'react-router-dom-v5-compat';
 import { useAsyncFn, useInterval, useMeasure } from 'react-use';
 
 import { GrafanaTheme2, urlUtil } from '@grafana/data';
+import { logInfo } from '@grafana/runtime';
```

Comment:
is this logInfo change intentional? is not correct to use the one from `'../../Analytics' `?

Reply:
No this seems like a mistake, I'll fix that.